tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/style.css

```diff
@@ -0,0 +1,288 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:200,300,400,500,600,700,800,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: poppins, sans-serif;
+}
+
+body {
+  background-color: lightblue;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+.container {
+  position: relative;
+  width: 100%;
+  margin: 50px;
+  max-width: 1000px;
+  min-height: 100px;
+  background-color: white;
+  display: grid;
+  grid-template-columns: 1.4fr 2fr;
+  box-shadow: 0 35px 55px rgba(0, 0, 0, 0.1);
+}
+
+.profiletext {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 20px;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.2);
+}
+
+.profiletext .imgbox {
+  position: relative;
+  width: 200px;
+  height: 200px;
+  border-radius: 50%;
+  overflow: hidden;
+}
+
+.profiletext .imgbox img {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  width: 200px;
+  height: 200px;
+}
+
+.container .left-side {
+  position: relative;
+  background: #003147;
+  padding: 40px;
+}
+
+.profiletext h2 {
+  color: white;
+  font-size: 1.5em;
+  margin-top: 20px;
+  text-transform: uppercase;
+  text-align: center;
+  font-weight: 600;
+  line-height: 1.4em;
+}
+
+.profiletext h2 span {
+  font-size: 0.8em;
+  font-weight: 300;
+}
+
+.contact {
+  padding: 40px;
+}
+
+.title {
+  color: white;
+  text-transform: uppercase;
+  font-weight: 600;
+  letter-spacing: 1px;
+  margin-bottom: 20px;
+}
+
+.contact ul {
+  position: relative;
+}
+
+.contact ul li {
+  list-style: none;
+  position: relative;
+  margin: 20px 0;
+  cursor: pointer;
+}
+
+.contact ul li .icon {
+  display: inline-block;
+  width: 20px;
+  font-size: 18px;
+  color: aquamarine;
+}
+
+.contact ul li span {
+  color: white;
+  font-weight: 300;
+}
+
+.contact.education li {
+  margin-bottom: 15px;
+}
+
+.contact.education h5 {
+  color: #03a9f4;
+}
+.contact.education h4:nth-child(2) {
+  color: white;
+  font-weight: 00;
```

Comment:
What is 00?

Reply:
yes😅, I will fix it asap @faran4engg @AlishaPrasad